### PR TITLE
X946 end user create work

### PR DESCRIPTION
**Pull request by seenanair**
* Create work request' link added to Dashboard
*  If user not logged in, clicking on 'Work request' link navigates to login page 
* If user logged in is end-user or above , access to SGP page is allowed
* For end-users only top portion of SGP page is visible
* For end-user, after successful creation of SGP a  message to  RNAscope/IHC template is displayed as well as a  link to SGP FRED page
* File manager menu item is visible and accessible to end-user
* If the user logged in is end-user, only the SGP numbers created by the user will be listed

### CODE REVIEW

File: src/components/workAllocation/WorkAllocation.tsx [hunk 1]
```diff
@@ -206,7 +211,24 @@ export default function WorkAllocation() {
...
+            {userRoleIncludes(UserRole.Enduser) && (
+              <div className={'flex flex-row text-pink-600 mt-4'}>
+                <InfoIcon className={`bg-white inline-block '}`} />
+                <a
+                  href={
+                    'https://fred.wellcomegenomecampus.org/Interact/Pages/Content/Document.aspx?id=6817&utm_source=interact&utm_medium=side_menu_category'
+                  }
+                  className="text-sp-600 hover:text-sp-700 font-semibold hover:underline text-base tracking-wide"
+                >
+                  Please complete RNAscope/IHC template for probes/antibody and fluorophore
```
Comment (khelwood) [inline src/components/workAllocation/WorkAllocation.tsx:226]: Can this be styled to look less like an error message please?

**Review by khelwood — COMMENTED** [review]
Should there be a link in the top bar for end users to go to the work request page?
Should the "active" checkbox **not** be ticked by default for end users, because it's confusing not to see the work number you've just created as an option.

File: src/components/workAllocation/WorkAllocation.tsx (hunk 1 above)
Comment (seenanair) [reply src/components/workAllocation/WorkAllocation.tsx:226]: Ok, More like an information I hope.

**Comment by seenanair** [comment]
> Should there be a link in the top bar for end users to go to the work request page? Should the "active" checkbox **not** be ticked by default for end users, because it's confusing not to see the work number you've just created as an option.

I will add. "SGP Management' in top menu
You are right about 'active' - I am not sure whether we need that 'active' box now. The checkbox was to  support upload only for 'active' work numbers  which was the initial case and later on changed to all.
I think the default can be 'active' box not to be turned on, for all type of users.

**Comment by khelwood** [comment]
Having it turned off by default makes sense.

File: src/components/workAllocation/WorkAllocation.tsx (hunk 1 above)
Comment (seenanair) [reply src/components/workAllocation/WorkAllocation.tsx]: Done

**Comment by seenanair** [comment]
> Having it turned off by default makes sense.

Done 👍

File: src/components/workAllocation/WorkAllocation.tsx [hunk 2]
```diff
@@ -215,15 +215,20 @@ export default function WorkAllocation() {
...
+                Please complete RNAscope/IHC template for probes/antibody and fluorophore
                 <a
                   href={
                     'https://fred.wellcomegenomecampus.org/Interact/Pages/Content/Document.aspx?id=6817&utm_source=interact&utm_medium=side_menu_category'
                   }
-                  className="text-sp-600 hover:text-sp-700 font-semibold hover:underline text-base tracking-wide"
+                  className="underline text-blue-600 hover:text-blue-800 font-semibold ml-1"
                 >
-                  Please complete RNAscope/IHC template for probes/antibody and fluorophore
+                  here
```
[the remark this answers is not in the document]
Comment (seenanair) [reply src/components/workAllocation/WorkAllocation.tsx:231]: That appears as a link to click - 

https://user-images.githubusercontent.com/55585488/234593729-e8137fc5-c934-4875-b71c-19069e19d22b.mov

**Review by khelwood — APPROVED** [review]

**Pull request merged**